wfe/web-front-end_test.go

```diff
@@ -284,12 +394,11 @@ func TestIssueCertificate(t *testing.T) {
 		responseWriter.Body.String(),
 		"{\"type\":\"urn:acme:error:malformed\",\"detail\":\"Error unmarshaling certificate request\"}")
```

Comment:
I believe this is `go fmt`s doing, unless you mean the non-indentation of the JSON, in that case not really sure.